drivers/SmartThings/philips-hue/profiles/4-button-remote.yml

```diff
@@ -0,0 +1,30 @@
+name: 4-button-remote
+components:
```

Comment:
@greens @tpmanley is this how we normally model buttons.  Looking briefly at Zigbee just judging on profile names it seems like we might typically use one component per button in addition to a "main" component.  In which case do we want an additional button only component here?

Reply:
It looks like we've been a bit inconsistent. For Matter we did it this way, and for Zigbee/Z-Wave we did it like you describe. Ideally we'd be consistent across all device types, but in this case we're aiming for consistency with Matter in particular since the Hue bridge can also be joined using Matter, so I'm inclined to leave it as is. Thoughts @greens @ctowns ?

Reply:
Yeah it's not consistent and we often get feedback about the "main" catchall button. One of the main (no pun intended) reasons we _did_ include it in some cases was because there used to be a bug where a component's capabilities would not function if the main component didn't also include that capability. It has been fixed, but left us with this confusing legacy.

I'd err on the side of not having a button component in "main" unless it's used as just one of the buttons, rather than the typical implementation, which is that it fires from any component button event.

Reply:
The button component in `main` maps to to Button 1 in a 1, 2, or 4 button remote config; it is not a catch-all button capability, it only maps to Button 1. And as Tom indicated, I did it this way to be consistent in presentation with the Matter integration, which was a goal outlined in the PRD.

Reply:
IMO the layout we did with matter is the best for the user, and I like that we follow it here. I am fine with the main component representing the first button.